### PR TITLE
Add Maintenance Alert

### PR DESCRIPTION
**Pull request by phillipahereza**
## Description

Add maintenance alert strip on openAFRICA above the nav bar

## Type of change

Please delete options that are not relevant.

- [x] New feature (non-breaking change which adds functionality)

## Screenshots
screencapture-192-168-33-60-5000-2018-03-11-15_46_07

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation

### CODE REVIEW

File: ckanext/openafrica/templates/header.html [hunk 1]
```diff
@@ -1,16 +1,24 @@
 {% ckan_extends %}
 {% block header_wrapper %}
 
+
   <div class="navbar">
     <div class="navbar-inner navbar-fixed-top">
+        <div class="alert alert-info alert-dismissible text-center" role="alert">
```
Comment (DavidLemayian) [inline ckanext/openafrica/templates/header.html]: Please check indentation. It looks off.

**Review by DavidLemayian — CHANGES_REQUESTED** [review]

**Comment by DavidLemayian** [comment]
I'd like the alert to look as such with the dismiss button at the far right of course:

36672917-22dee4da-1b12-11e8-90cb-fdd561b5708d

**Comment by DavidLemayian** [comment]
Is there a "notification" system in CKAN that perhaps takes advantage of sessions to avoid showing the alert if it's already been dismissed once?

**Comment by phillipahereza** [comment]
There's no notification system, but I could use the browser sessionStorage using js

**Comment by DavidLemayian** [comment]
@phillipahereza There are conflicts and what's with the text shadow?

**Comment by DavidLemayian** [comment]
@phillipahereza The dates here need to change..

**Comment by DavidLemayian** [comment]
@phillipahereza Let's update the screenshot so we can see what it looks like.

You'll also notice that the language specifies GMT and not EAT in the screenshot I shared. And the specific timeline. Let's change this.

**Pull request merged**